Documentation/Configuration/Damage/Tutorial.md

```diff
@@ -0,0 +1,49 @@
+
+# Tutorial Damage
+
+<kbd>  admintoolbox_tutorial_dmg_allowed  </kbd>  
+<kbd>  List  ➞  -1  </kbd>
+
+<br>
+<br>
+
+## Description
+
+*Sets what types of damage the  `TUTORIAL`  role can take.*
+
+<br>
+<br>
+
+## Choices
```

Comment:
This makes it seem like there is only 2 choices. In reality, it can be any combination of the numbers here: https://github.com/ServerMod/Smod2/wiki/Enum-Lists#damagetype